
/bugbase_bringup:
  joy_node:
    ros__parameters:
      autorepeat_rate: 50
  teleop_node:
    ros__parameters:
      enable_button: 6
      axis_linear.x: 1
      scale_linear.x: 1.0
      axis_angular.yaw: 2
      scale_angular.yaw: 2.0
      

  odrive_node:
    ros__parameters:
      odrive_watchdog_enable: True
      odrive_watchdog_timeout: 0.5
      antislip_enable: False